src/common/transformations/include/transformations/common_optimizations/mark_subgraphs_to_keep_in_mixed_precision.hpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/pass/graph_rewrite.hpp"
+#include "openvino/pass/pass.hpp"
+#include "transformations_visibility.hpp"
+
+namespace ov {
+namespace pass {
+
+class TRANSFORMATIONS_API MarkSugraphsToKeepInMixedPrecision;
+
+// through this nodes precision sensitiveness is propagated
+extern std::shared_ptr<Node> propagate_through_ops;
```

Comment:
why do we need it? Looks not C++ approach.

Reply:
we needed this list of ops in 2 different places:  `mark_subgraphs_to_keep_in_mixed_precision` and `mark_exp_reduceop_to_keep_in_mixed_precision`. For historical reason (second file was introduced much later) values are set in the first file, to use them in the second file i redefined them in a header with `extern`. I could've move the not only definition but setting values into header as well but i opted to keep list to the file where list is most extensively used. 

Anyway if we collect all markings into a single file then this `extern` will not be needed any longer

Reply:
combined all 3 marking passed in a single file, no need for this `extern` anymore, removed it